.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: android
+
+jdk:
+  - openjdk8
+
+script:
+  - ./gradlew clean check
+
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" > "$ANDROID_HOME/licenses/android-sdk-license"
```

Comment:
Is this required? I thought Travis accepts all requested licenses [automatically](https://docs.travis-ci.com/user/languages/android/#dealing-with-licenses)?

Reply:
Cribbing from another project.  The link you pasted appears to require using the `trusty` dist.  Can explore in a follow-up.